contracts/token/ERC721/ERC721Metadata.sol

```diff
@@ -14,12 +14,12 @@ contract ERC721Metadata is Initializable, Context, ERC165, ERC721, IERC721Metada
     // Token symbol
     string private _symbol;
 
-    // Base URI
-    string private _baseURI;
-
     // Optional mapping for token URIs
     mapping(uint256 => string) private _tokenURIs;
 
+    // Base URI
+    string private _baseURI;
```

Comment:
We have to update the gap at the end of the contract. I _think_ it's only one slot that's being added here.

Reply:
Good catch, fixed! I'm reviewing the [changes from 2.5](6e901efa224dbeafa6df18d4e9c55c16889cb8f3) to make sure there is no other gap missing.

Reply:
No other storage changes. We should be good to release 2.5 eth-pkg with this!